OSDOCS 10197: Added the illustration of enclave support and explained the diagram. (#81930)
openshift/openshift-docs: modules/oc-mirror-enclave-support-about.adoc

* You can mirror content directly from the centralized internal registry to enclaves without restarting the mirroring process from the internet for each enclave.

* You can minimize data transfer between network stages, so to ensure that a blob or image is transferred only once from one stage to another.

> == How to mirror to an enclave

Reviewer: Also I think this title might be a little misleading because this section is followed by a page for "mirroring to an enclave", which sounds like a similar concept to "how to mirror an enclave". I don't know what a good alternative would be, but I would suggest you think of one to replace this heading with. Here are a few ideas, though I don't know how accurate they'd be:

```suggestion
== Enclave mirroring workflow
```

```suggestion
== About enclave mirroring
```

image::445_OpenShift_Enclave_support_0724.png[Enclave Support]

> The above illustration outlines the flow for using the oc-mirror plugin in different environments, including environments with and without an internet connection.

Reviewer: In [this section](https://redhat-documentation.github.io/supplementary-style-guide/#accessibility-visual-info) of the SSG, we're advised to avoid indicating direction such as above and below. In this case, you'd want to use "previous" instead. Also, "image" is a simpler word imo and the one used most often throughout our docs:
```suggestion
The previous image outlines the flow for using the oc-mirror plugin in different environments, including environments with and without an internet connection.
```

*Environment with Internet Connection*:

> - The user executes oc-mirror plugin V2 to mirror content from an online registry to a local disk directory.
> - The mirrored content is saved to the disk for transfer to offline environments.

Reviewer: So in this case I think that because this flow implies an order of actions (as in, you must take the first action before you can take the second), ordered/numbered list items would be more appropriate here:
```suggestion
. The user executes oc-mirror plugin V2 to mirror content from an online registry to a local disk directory.
. The mirrored content is saved to the disk for transfer to offline environments.
```

See [this section](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#ordered-lists) of our ISG for more info.

> *Airgapped Enterprise Environment (No Internet)*:

Reviewer: @sherine-k and @lmzuccarelli can "airgapped" be replaced with "disconnected" here? Per our [repo's term glossary](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#disconnected), "disconnected environment" is always preferred over "restricted" and "air-gapped", and we generally try to stay consistent with our wording.

In this case, I think that because "air gapped" is already used in the image above, it's not worth changing in this PR, but if "disconnected" is at all interchangeable with "air gapped" in this context, I think this is a fix we'll want to make down the line.

Author: Let me change the phrase "airgapped" to "disconnected".

Reviewer: Sounds good, just keep in mind that at some point, the image needs to be updated to remove "airgapped" as well, and that right now there's a sort of awkward discrepancy in the phrasing.

> -  Flow 1:

Reviewer: So I am not sure whether Flow 1 and Flow 2 are two different options that users can choose to use, or something that users have to do in order. If they both need to be done, and in order, I would make this line and [line 41](https://github.com/openshift/openshift-docs/pull/81930/files#diff-50b8a98363a7e5aa947d524510775ee4db48b65b89c6206ffa7bee64678c609aR41) a numbered list item.

If they are not flows that need to be done in order, then they can stay as unordered list items, but I would suggest replacing the hyphen with a * instead. This would just follow the way our docs typically do things a bit more closely:
```suggestion
*  Flow 1:
```

[id="oc-mirror-enclave-how-to_{context}"]
== Enclave mirroring workflow

image::445_OpenShift_Enclave_support_0724.png[Enclave Support]

The previous image outlines the flow for using the oc-mirror plugin in different environments, including environments with and without an internet connection.

*Environment with Internet Connection*:

. The user executes oc-mirror plugin v2 to mirror content from an online registry to a local disk directory.
. The mirrored content is saved to the disk for transfer to offline environments.

*Disconnected Enterprise Environment (No Internet)*:

*  Flow 1:

** The user runs oc-mirror plugin v2 to load the mirrored content from the disk directory, which was transferred from the online environment, into the `enterprise-registry.in` registry.

* Flow 2:

.. After updating the `registries.conf` file, the user executes the oc-mirror plugin v2 to mirror content from the `enterprise-registry.in` registry to an enclave environment.

.. The content is saved to a disk directory for transfer to the enclave.

*Enclave Environment (No Internet)*:

> . The user runs oc-mirror plugin v2 to load content from the disk directory into the `enclave-registry.in` registry.

Reviewer: It was good to change this from the hyphen format, but I think that because this list has only a single item, there is no possible order to follow, in which case this can be an unordered bullet:
```suggestion
* The user runs oc-mirror plugin v2 to load content from the disk directory into the `enclave-registry.in` registry.
```
